### PR TITLE
BUG - Version Mismatch Between ChromeDriver and Installed Chrome Browser

### PR DESCRIPTION
**Pull request by IsraelleHub**
## Description of Change
The error we are encountering is due to a version mismatch between the ChromeDriver and the installed Chrome browser. 
We need to install a ChromeDriver version that matches our Chrome browser version (126.0.6478.182) and specify its path when running the axe command.
[Error Link](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/10177268188)

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: .github/workflows/documentation_accessibility_checks.yml
Comment (timwright12) [inline .github/workflows/documentation_accessibility_checks.yml:72]: Is this OK to remove?

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: .github/workflows/documentation_accessibility_checks.yml
Comment (IsraelleHub) [reply .github/workflows/documentation_accessibility_checks.yml:12]: Yes, I am waiting for final review to remove it.

File: .github/workflows/documentation_accessibility_checks.yml
Comment (IsraelleHub) [reply .github/workflows/documentation_accessibility_checks.yml:72]: Yes. Now it is using start_slack_thread.yml instead. It is ok. Other Workflow are using the same format as well.

Comment (timwright12) [reply .github/workflows/documentation_accessibility_checks.yml:72]: Can you break this work off into another PR if it isn't related to the chromium issue?

File: .github/workflows/documentation_accessibility_checks.yml
Comment (timwright12) [reply .github/workflows/documentation_accessibility_checks.yml:12]: My mistake, I thought I read that this was ready for review

Comment (IsraelleHub) [reply .github/workflows/documentation_accessibility_checks.yml]: It is ready for review. it's ready for review

File: .github/workflows/documentation_accessibility_checks.yml
Comment (IsraelleHub) [reply .github/workflows/documentation_accessibility_checks.yml]: yes, I rolled it back

**Review by timwright12 — APPROVED** [review]

**Pull request merged**
